src/main/java/frc/trigon/robot/subsystems/arm/ArmCommands.java

```diff
@@ -0,0 +1,84 @@
+package frc.trigon.robot.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.DeferredCommand;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+import java.util.Set;
+
+public class ArmCommands {
```

Comment:
Everything here needs to be static, you don't want to create a new ArmCommands every time.